projects/fsastorefrontstyles/scss/components/cms/_find-agent-search-list.scss

```diff
@@ -57,9 +57,6 @@
     .fas {
       font-size: var(--cx-h3-font-size);
     }
-    .fa-phone {
```

Comment:
Removed this code since I upgraded fontawesome lib and now can use an icon which doesn't require to be rotated.